Chore: Generate docs for project-policies (#161)
env0/terraform-provider-env0: docs/data-sources/project_policy.md

---
# generated by https://github.com/hashicorp/terraform-plugin-docs
page_title: "env0_project_policy Data Source - terraform-provider-env0"
subcategory: ""
description: |-
  
---

# env0_project_policy (Data Source)

<!-- schema generated by tfplugindocs -->
## Schema

### Required

- **project_id** (String) id of the project

### Read-Only

> - **disable_destroy_environments** (Boolean) disable destroy environments

Reviewer: ```suggestion
- **disable_destroy_environments** (Boolean) Disallow destroying environment in the project
```

Author: 6abf47e40681ce2f4337a73c8107fc896bc37c38

> - **id** (String) id of the policy

Reviewer: Can we make this the first item in this section?

Author: I can change it manually, but the next time we run `generateDocs` it will change it. Seems like it arranges it alphabetically. Couldn't find any flag/config in the plugin [repo](https://github.com/hashicorp/terraform-plugin-docs)

Reviewer: So forget about it, it's not really important.

> - **include_cost_estimation** (Boolean) include cost estimation

Reviewer: ```suggestion
- **include_cost_estimation** (Boolean) Enable cost estimation for the project
```

Author: f4bf0e16d33b1202f45ff99f268d0fc9df414f67

> - **number_of_environments** (Number) number of environments per project

Reviewer: ```suggestion
- **number_of_environments** (Number) Max number of environment a single user can have in this project
```

Author: 24e31dfc6889b1525eac6d73e7236923febd39b9

> - **number_of_environments_total** (Number) number of environments total

Reviewer: ```suggestion
- **number_of_environments_total** (Number) Max number of environments in this project
```

Author: 9e9d69398d7a2c3e51511a510c3f9a69e1726fd6

> - **requires_approval_default** (Boolean) requires approval

Reviewer: ```suggestion
- **requires_approval_default** (Boolean) Requires approval default value when creating a new environment in the project
```

Author: d094b4cc5c89f4da24007538620b48ef741264b2

> - **skip_apply_when_plan_is_empty** (Boolean) skip apply when plan is empty

Reviewer: ```suggestion
- **skip_apply_when_plan_is_empty** (Boolean) Skip apply when plan has no changes
```

Author: d09e045a088aab9f3c06bf8848b6bf0c26246312

> - **skip_redundant_deployments** (Boolean) skip redundant deployments

Reviewer: ```suggestion
- **skip_redundant_deployments** (Boolean) Automatically skip queued deployments when a newer deployment is triggered
```

Author: 8dbb1352c52d3bff3394d83669d8ac6623bd9539
